condor/condor_meter

```diff
@@ -551,11 +551,14 @@ def determine_host_description(classad):
 
     If there's no special host description, this returns None.
     """
-    if 'MATCH_EXP_JOBGLIDEIN_ResourceName' not in classad:
+    try:
+        if classad['MATCH_EXP_JOBGLIDEIN_ResourceName'] == 'Local Job':
+            host_descr = GratiaCore.Config.get_SiteName()
+        else:
+            host_descr = classad['MATCH_EXP_JOBGLIDEIN_ResourceName']
+    except KeyError:
```

Comment:
I'm not really attached to any particular way of doing it but I think it's safe in this case:

* If `get_SiteName()` throws a `KeyError`, is it worthy of stopping the entire probe run or should we just continue constructing the record as best we can? We could try setting the host description to `MATCH_EXP_JOBGLIDEIN_ResourceName`, which would be `Local Job` in that case, but I'm not sure if that's any better than setting it to `None` (and possibly worse because of how the `None` case is [handled](https://github.com/opensciencegrid/gratia-probe/blob/master/condor/condor_meter#L704-L708)).
* If `classad[]` throws a `KeyError`, we have no hope of knowing where the job ran so we should return `None`. We could feasibly set it to the ProbeConfig's SiteName but it wouldn't necessarily reflect reality.